native/cocos/renderer/GFXDeviceManager.h

```diff
@@ -139,6 +128,17 @@ class CC_DLL DeviceManager final {
     }
 
 private:
+    static void addSurfaceEventListener() {
```

Comment:
So `addSurfaceEventListener` will never be invoked now?
If so, could it be removed?

Reply:
It is used invoked in DeviceManager internally.